interactions/client/get.py

```diff
@@ -0,0 +1,37 @@
+from typing import Type, TypeVar
+
+from ..api.models.guild import Guild
+from ..api.models.message import Emoji
+from ..api.models.role import Role
+from .bot import Client
+
+_T = TypeVar("_T")
+
+
+async def get(client: Client, obj: Type[_T], **kwargs) -> _T:
+    r"""
+    A helper method for retrieving data from the Discord API in its object representation.
+
+    :param client: Your instance of `interactions.Client`
+    :type client: Client
+    :param obj: The object to get. Should be a class object (not an instance!). For example: `interactions.Channel`.
+    :type obj: Type[_T]
+    :param \**kwargs: The arguments to pass to the HTTP method.
+    :type \**kwargs: dict
+    :return: The object we're trying to get.
+    :rtype: object
+    """
+
+    if not isinstance(obj, type):
+        raise TypeError("The object must not be an instance of a class!")
+
+    _name = f"get_{obj.__name__.lower()}"
+
+    if obj in (Role, Emoji):
+        _guild = Guild(**await client._http.get_guild(kwargs.pop("guild_id")), _client=client._http)
+        _func = getattr(_guild, _name)
+        return await _func(**kwargs)
+
+    _func = getattr(client._http, _name)
```

Comment:
Remove return and await, put else on 34, indent second _func to inside else block

Reply:
The func for role and emoji are returning a full object tho so it makes sense to return directly from it

Reply:
Ohh ok